### ISSUE TITLE
Please delete this bug report.

### ISSUE THREAD

**Issue by MemoryHax** [xADDBx/ToyBox-Wrath#21]
Please delete this bug report.

**Comment by Evilvonbek** [comment]
split it up into paragraphs ue, it's hard as hell to read that the way it is

**Comment by xADDBx** [comment]
1. Can you reproduce this issue with only ToyBox installed?
2. Could you share the steps you do to actually cause the issue (preferably with a save just before you encounter the issue)?

**Comment by MemoryHax** [comment]
My bad.  Give me a little time to test it again with the other mods disabled.  I will break the message up in paragraphs.  I didn't want to hit "enter" and send the message prematurely like some other chat or blog windows.  I hate when that happens.  I will be back after I do some testing.  Thank you for your patience, and your help.

**Comment by MemoryHax** [comment]
Okay.  I tried the game with only updated Toybox/Game/UMM on (no other mods) and took screenshots to show the characters disappearing when they are talking outside of my party.  While in my party, I see them.  Cannot click the weapon chest still.  This occurs in the very first cave when you fall through the ground after Lord of Locusts destroys the road/sidewalk.  You will see the characters switch off being visible.

![Screenshot (1096)](https://github.com/user-attachments/assets/c53a40e2-7d94-45df-969a-724ad4fa02d9)
![Screenshot (1098)](https://github.
![Screenshot (1101)](https://github.com/user-attachments/assets/85df9e0d-03a6-4dcf-8dc2-3bfe19bacff5)
com/user-attachm
![Screenshot (1102)](https://github.com/user-attachments/assets/b4d52271-09d2-4724-a8ef-3ffd0c6fb08d)
ents/assets/d516bd44-74a7-4bab-881f-98c41d7784a1)
![Screenshot (1105)](https://github.com/user-attachments/assets/911916cd-8db9-48de-927a-5784c2c328f5)
![Screenshot (1106)](https://github.com/user-attachments/assets/987414c2-b7b9-4d1b-a65d-a0963eb384df)
[Manual_4_Underground_Caves_of_Kenabres__15_Arodus__VIII__4715__12_06_04.zip](https://github.com/user-attachments/files/16624250/Manual_4_Underground_Caves_of_Kenabres__15_Arodus__VIII__4715__12_06_04.zip)

**Comment by xADDBx** [comment]
What options/settings do you have enabled in ToyBox?

**Comment by MemoryHax** [comment]
I have checked and unchecked many features over time, hoping that I could find the source of my issue.  First, I unchecked many features that had to do with highlighting items or making visual changes to the screen.  The item and character disappearing still happened.  Then I unchecked every feature that had anything to do with UI, color, markers, etc.  No luck.  Last, I tried having only leveling and stat changes on.  Still I have an issue.  Now, I think I will uncheck everything, and see if it still makes things disappear.  I will report back to you again soon.
